manifests/h/HeidelbergUniversity/GaiaSky/3.5.1/HeidelbergUniversity.GaiaSky.locale.en-US.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: HeidelbergUniversity.GaiaSky
+PackageVersion: 3.5.1
+PackageLocale: en-US
+Publisher: Zentrum für Astronomie der Universität Heidelberg

```

```suggestion
Publisher: ARI - ZAH - Universität Heidelberg
```
This should match Control Panel